ext/nnstreamer/tensor_filter/tensor_filter_tensorflow.cc

```diff
@@ -192,8 +193,8 @@ TFCore::getModelPath ()
 /**
  * @brief	the definition of a deallocator method
  */
-static void
-DeallocateBuffer (void* data, size_t t) {
+void
+TFCore::releaseBuffer (void* data, size_t t) {
   std::free (data);
```

Comment:
This comment is not related to this PR, but how about using g_free here for the consistency of the allocator/deallocator pair.  In the near future, we can replace those g_file_test and g_file_get_contents functions with c++ styles.